general/app_releases/v5/v5.0.0.md

```diff
@@ -0,0 +1,133 @@
+- [MOBILE-4678](https://moodle.atlassian.net/browse/MOBILE-4678) - Courses are not displayed as "available offline/downloaded" after bulk downloaded unless the user clicks on the three dots menu
+- [MOBILE-4687](https://moodle.atlassian.net/browse/MOBILE-4687) - Weird keyboard behaviour when creating comments
+- [MOBILE-4688](https://moodle.atlassian.net/browse/MOBILE-4688) - Files whose name contains % (percent, percentage) are not downloaded
+- [MOBILE-4690](https://moodle.atlassian.net/browse/MOBILE-4690) - Quiz configured for offline with deferred feedback is not displaying question validation messages
+- [MOBILE-4692](https://moodle.atlassian.net/browse/MOBILE-4692) - Revise warning messages displayed in forums
+- [MOBILE-4694](https://moodle.atlassian.net/browse/MOBILE-4694) - App Behat tests no longer compatible with PHP 7.4
+- [MOBILE-4698](https://moodle.atlassian.net/browse/MOBILE-4698) - Description not displayed on the Feedback module page in the Moodle App
+- [MOBILE-4700](https://moodle.atlassian.net/browse/MOBILE-4700) - Feedback: Review size of labels + some questions disappeared
+- [MOBILE-4704](https://moodle.atlassian.net/browse/MOBILE-4704) - Exit from full screen mode if user navigates
+- [MOBILE-4705](https://moodle.atlassian.net/browse/MOBILE-4705) - RecordRTC does not play when added to a file resource
+- [MOBILE-4706](https://moodle.atlassian.net/browse/MOBILE-4706) - RecordRTC videos aren't reproduced in full (just first few seconds)
+- [MOBILE-4707](https://moodle.atlassian.net/browse/MOBILE-4707) - Users cannot access attempts review when a quiz is closed
+- [MOBILE-4708](https://moodle.atlassian.net/browse/MOBILE-4708) - In certain configurations 'This Quiz has offline data to be synchronised' is displayed incorrectly
+- [MOBILE-4715](https://moodle.atlassian.net/browse/MOBILE-4715) - If you disable blog for the app, the link to blogs from the activities still appears
+- [MOBILE-4716](https://moodle.atlassian.net/browse/MOBILE-4716) - data-open-in attribute in links no longer working with app 4.5
+- [MOBILE-4720](https://moodle.atlassian.net/browse/MOBILE-4720) - If you only have available a custom field as a course filter, then the filter is empty (nothing is preselected)
+- [MOBILE-4722](https://moodle.atlassian.net/browse/MOBILE-4722) - Bug in iOS 18+ due to deprecation in cordova-plugin-inappbrowser
+- [MOBILE-4723](https://moodle.atlassian.net/browse/MOBILE-4723) - Disabling all blocks makes myoverview disappear (4.4 regression)
+- [MOBILE-4728](https://moodle.atlassian.net/browse/MOBILE-4728) - Close button doesn't have good contrast
+- [MOBILE-4732](https://moodle.atlassian.net/browse/MOBILE-4732) - READ_MEDIA_IMAGES and READ_MEDIA_VIDEO permissions are still present
+- [MOBILE-4743](https://moodle.atlassian.net/browse/MOBILE-4743) - If the template contains { or }, it will not be displayed as an error.
```

Comment:
This line is making the page crash because of the "{ or }" part. We can rename it to "curly brackets" in these docs.

```suggestion
- [MOBILE-4743](https://moodle.atlassian.net/browse/MOBILE-4743) - If the template contains curly brackets, it will not be displayed as an error.
```